rest/views.py

```diff
@@ -48,6 +49,39 @@
 from django.http.response import Http404
 
 
+def get_urls(raw_urls, list_urls, urlbase=''):
+    """
+    Recursively builds a list of all the urls in the current project and
+    the name of their associated view
+    """
+    list_urls = list_urls or []
+    from operator import itemgetter
+    for entry in raw_urls:
+        full_url = (urlbase + entry.regex.pattern).replace('^', '')
```

Comment:
I actually like the idea of pulling all the URLs automatically from the URL definition. Is there a better way to do this?